sensor/kubernetes/clusterhealth/updater.go

```diff
@@ -223,6 +223,8 @@ func (u *updaterImpl) getLocalScannerInfo() *storage.ScannerHealthInfo {
 		}
 	}
 
+	// TODO/MC: Extend for ScannerV4 case?
```

Comment:
Now that you've created a ticket do we still need that comment?

Reply:
right, removed.